### DISCUSSION TITLE
Missing service versions in dashboards

### DISCUSSION THREAD

**Discussion by bombillazo** [supabase/supabase#32974] [Feature Requests]
# Bug report

- [x] I confirm this is a bug with Supabase, not with my own application. 
- [x] I confirm I have searched the [Docs](https://docs.supabase.com), GitHub [Discussions](https://github.com/supabase/supabase/discussions), and [Discord](https://discord.supabase.com). 

## Describe the bug

The dashboard does not give a complete description of the project stack versions. It give only a partial list of services.

This is important to know if we want to have version parity across our local development as well as our staging/production environments.

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Go to project settings -> Infrastructure

## Expected behavior

We'd like to know what versions are running in our project for:
- Storage
- Edge Runtime
- Pooler
- Kong

## Screenshots

<img width="1121" alt="image" src="https://github.com/supabase/supabase/assets/44002321/84a02786-1b0e-473d-9f86-8678518a6717">

## System information

- OS: [e.g. macOS, Windows]
- Browser (if applies) [e.g. chrome, safari]
- Version of supabase-js: [e.g. 6.0.2]
- Version of Node.js: [e.g. 10.10.0]

**Comment by joshenlim** [comment]
Just FYI - gonna change the label from bug to enhancement as this isn't a bug
for transparency, we're aware that not all service versions are available in the dashboard and are also looking into making those information available in the UI

**Comment by robsalsa** [comment]
> Just FYI - gonna change the label from bug to enhancement as this isn't a bug for transparency, we're aware that not all service versions are available in the dashboard and are also looking into making those information available in the UI

Hey was someone working on this previously or currently working on it? Or is there gonna be a new fix for this issue because my group and I wanted to work on this?

**Comment by Hallidayo** [comment]
Hi everyone, due to inactivity on this issue I've moved the issue over to discussions/enhancements.
